### PR TITLE
Using stable version of symfony/polyfill-php80

### PR DESCRIPTION
**Pull request by gitromba**

### CODE REVIEW

**Review by Alroniks — APPROVED** [review]
Looks good

File: composer.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "monolog/monolog": "^1.23.0",
     "fluent/logger": "^1.0.1",
     "symfony/lock": "^3.4",
-    "symfony/polyfill-php80": "1.21.x-dev",
+    "symfony/polyfill-php80": "^1.20",
```
Comment (gabrielfs7) [inline composer.json:70]: This new function will cause migration issues when moving to php8:

https://github.com/oat-sa/generis/commit/e4e843d6a0207654e2f59975070559f74afce977#diff-0c5f5d447957a7012b605c9fea0336c1f61017340a68d0797fa0d789be48f74cR165

@Alroniks Do you think we should stick to that?

**Comment by gitromba** [comment]
I've replaced the `str_starts_with` calls with native implementation. Unit tests are passing. @gabrielfs7 @Alroniks FYI

**Review by gabrielfs7 — APPROVED** [review]
Code LGTM. I did a quick test locally and could not find any other usage of `str_starts_with` in our codebase.

Thanks @gitromba

**Pull request merged**